.github/workflows/choreolib.yml

```diff
@@ -158,6 +184,24 @@ jobs:
       - name: Combine
         run: ./gradlew publish -Pthirdparty
         working-directory: combiner
+
+      - name: Transfer ChoreoLib Artifacts
+        if: startsWith(github.ref, 'refs/tags/v')
+        run: |
+          cp -r ~/releases/maven/development/choreo/* ChoreoLib/dep/choreo
+          pushd ./ChoreoLib
```

Comment:
Is this `pushd` necessary? It's effectively just a `cd` command, and I was under the impression subsequent actions have independent shell contexts.

Reply:
No. I'll remove it.